City_CRUD_API/crud.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy.future import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from City_CRUD_API import models
+from City_CRUD_API.schemas import CityCreate
+
+
+async def get_all_cities(db: AsyncSession):
+    result = await db.execute(select(models.City))
+    return result.scalars().all()
+
+
+async def create_city(db: AsyncSession, city: CityCreate):
+    db_city = models.City(
+        name=city.name,
+        additional_info=city.additional_info,
+    )
+    db.add(db_city)
+    await db.commit()
+    await db.refresh(db_city)
+
+    return db_city
```

Comment:
Consider adding error handling for the `create_city` function. If the city creation fails due to database constraints or other issues, it would be beneficial to catch exceptions and return a meaningful error message.